ui/page/transaction/transactions_page.go

```diff
@@ -271,125 +310,44 @@ func (pg *TransactionsPage) loadTransactions(wal sharedW.Asset, offset, pageSize
 	return txs, len(txs), err
 }
 
+func settingCommonDropdown(t *cryptomaterial.Theme, drodown *cryptomaterial.DropDown) {
```

Comment:
`settingCommonDropdown` is not clear enough, I think. Nit: drodown

Reply:
what is your suggestion?